homework3/task03/tests/test_filter.py

```diff
@@ -0,0 +1,130 @@
+from project_filter.filter import Filter, make_filter
+
+
+def test_apply_positive_even():
+    positive_even = Filter(
+        lambda a: a % 2 == 0, lambda a: a > 0, lambda a: isinstance(a, int)
+    )
+    assert positive_even.apply(range(10)) == [2, 4, 6, 8]
+
+
+def test_apply_sum_with_7_is_zero():
+    sum_with_7_is_zero = Filter(lambda a: a != 0, lambda a: a + 7 == 0)
```

Comment:
Is any logical relation between this functions `lambda a: a != 0`  `lambda a: a + 7 == 0`?

Reply:
Yes, you are right, we can choose only the second one.